chain/network/src/routing_table_actor.rs

```diff
@@ -14,22 +16,355 @@ use crate::ibf::{Ibf, IbfBox};
 use crate::ibf_peer_set::IbfPeerSet;
 #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
 use crate::ibf_set::IbfSet;
-use crate::routing::Edge;
+use crate::routing::{Edge, EdgeType, Graph, SAVE_PEERS_MAX_TIME};
 #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
 use crate::routing::{SimpleEdge, ValidIBFLevel, MIN_IBF_LEVEL};
 use crate::types::StopMsg;
 #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
-use crate::types::{PartialSync, RoutingState, RoutingVersion2};
+use crate::types::{PartialSync, PeerMessage, RoutingState, RoutingSyncV2, RoutingVersion2};
+#[cfg(feature = "delay_detector")]
+use delay_detector::DelayDetector;
+use near_primitives::utils::index_to_bytes;
+use near_store::db::DBCol::{ColComponentEdges, ColLastComponentNonce, ColPeerComponent};
+use near_store::{Store, StoreUpdate};
+use std::sync::Arc;
+use std::time::{Duration, Instant};
 
-/// Actor that maintains routing table information.
-/// TODO (PIOTR, #4859) Finish moving routing table computation to new thread.
-#[derive(Default)]
+#[derive(Debug, Eq, PartialEq)]
+pub enum Prune {
+    Allow,
```

Comment:
can you add comment explaining the difference between Allow and Force

Reply:
I renamed variants to become self explanatory:
```
pub enum Prune {
    PruneOncePerHour,
    PruneNow,
    Disable,
}
```